docs/docs-content/clusters/cluster-management/backup-restore/backup-restore.md

```diff
@@ -94,6 +94,31 @@ or
 [Add a Backup Location using Dynamic Credentials](/clusters/cluster-management/backup-restore/add-backup-location-dynamic)
 guide.
 
+## Volume Snapshots
+
+<!-- prettier-ignore -->
+You can include [volume snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) in your cluster backups if the Container Storage Interface (CSI) driver you are using supports them. For more information about volume support, check the CSI pack README for your CSI driver. You can find a list of all Packs in the <VersionedLink text="Packs List" url="/integrations/" />
+page.
+
+<!-- prettier-ignore -->
+If the CSI driver supports volume backup, check if the CSI driver includes the snapshot controller, the required volume snapshot class, and the required Custom Resource Definitions (CRD). Otherwise, ensure the cluster profile includes the <VersionedLink text="Volume Snapshot Controller" url="/integrations/packs/?pack=volume-snapshot-controller" /> pack
+included. Use version 8 or greater of the pack. The Volume Snapshot Controller pack is required to take snapshots of volumes. The Volume Snapshot Controller also includes the required volume snapshot class. Check out the [Add a Pack](../../../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-pack-addon.md) guide to learn how to add a pack to a cluster profile.
+
```

Comment:
:::tip

For clusters running Virtual Machine Orchestrator, the VMO pack has built-in support for Snapshot Controller and includes the VolumeSnapshot.

Reply:
I'm going to add a VMO section. I think it would be good to emphasize to still review the CSI README 😄